app/perfaddresscache.h

```diff
@@ -89,7 +90,7 @@ class PerfAddressCache
     SymbolCacheEntry findSymbol(const QByteArray &filePath, quint64 relAddr);
 
     /// extract all symbols in @p module into a structure suitable to be passed to @p setSymbols
-    static SymbolCache extractSymbols(Dwfl_Module *module, quint64 elfStart, bool isArmArch);
+    static SymbolCache extractSymbols(Dwfl_Module* module, quint64 elfStart, PerfRegisterInfo::Architecture arch);
```

Comment:
the `*` location should remain as before

Reply:
Oh, I was using the one from hotspot (normally checked out in a parent directory, so clang-format would find it). I guess we could make a copy here, or maybe just update the one in hotspot?